src/main/scala/org/zalando/spearheads/innkeeper/utils/HttpClient.scala

```diff
@@ -7,7 +7,7 @@ import akka.http.scaladsl.model.{ HttpHeader, HttpMethod, HttpMethods, HttpReque
 import akka.stream.ActorMaterializer
 import com.google.inject.Inject
 import org.slf4j.LoggerFactory
-import spray.json._
+import spray.json.{ JsValue, pimpString }
```

Comment:
I didn't chose the weird name :)) it's for `"{}".parseJson`

On Tue, Jan 26, 2016 at 4:41 PM, Moritz Grauel <notifications@github.com>
wrote:

> In src/main/scala/org/zalando/spearheads/innkeeper/utils/HttpClient.scala
> <https://github.com/zalando/innkeeper/pull/36#discussion_r50851281>:
>
> > @@ -7,7 +7,7 @@ import akka.http.scaladsl.model.{ HttpHeader, HttpMethod, HttpMethods, HttpReque
> >  import akka.stream.ActorMaterializer
> >  import com.google.inject.Inject
> >  import org.slf4j.LoggerFactory
> > -import spray.json._
> > +import spray.json.{ JsValue, pimpString }
>
> pimpString?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zalando/innkeeper/pull/36/files#r50851281>.
>